src/styles/containerComposeStyles/types.ts

```diff
@@ -0,0 +1,5 @@
+import {ViewStyle} from 'react-native';
+
+type ContainerComposeStyles = ViewStyle[];
```

Comment:
Isn't this should be `StyleProp<ViewStyle>`?

And so, we probably don't have to create a shared style for `containerComposeStyles`

Reply:
I updated the PR, is that what you meant? 😄